### PR TITLE
implement GM Prompt option for dex scan

### PR DESCRIPTION
**Pull request by Ellamental2**
enabled the GM Prompt option for pokedex scan, this will now give the following prompt to the GM. #139 

image

If no is selected the player will be informed that they are unable to use their pokedex:
image

if nothing is selected or the dialog is closed it will warn the player that their request timed out:
image

### CODE REVIEW

**Comment by Ellamental2** [comment]
on timeout the below message will be sent to gm only
image
The buttons can then be used to show the user the requested dex entry

**Review by dylanpiera — COMMENTED** [review]
I'm gonna make some CSS changes to this but besides that looks good.

File: lang/en.json [hunk 1]
```diff
@@ -223,5 +223,13 @@
     "PTU.AutomationRangeIncrease.NONE":"None",
     "PTU.AutomationRangeIncrease.EFFECT_RANGE":"Effect Range",
 
-    "SETTINGS.ptu": ""
+    "SETTINGS.ptu": "",
```
Comment (dylanpiera) [inline lang/en.json:226]: ??

File: lang/en.json [hunk 2]
```diff
@@ -223,5 +223,13 @@
...
+
+    "PTU.DexScan.Off":"DM has turned off the Pokédex Scan feature.",
+    "PTU.DexScan.Description":"Description Only",
+    "PTU.DexScan.Full":"Full Scan",
+    "PTU.DexScan.Denied":"You are unable to use your Pokédex at this time.",
+    "PTU.DexScan.Timeout":"Request to DM for Pokédex Scan timed out.",
+    "No": "No",
+    "Yes": "Yes"
```
Comment (dylanpiera) [inline lang/en.json:234]: I feel like this may already exist in core foundry

**Review by dylanpiera — CHANGES_REQUESTED** [review]
As I said in previous comment ima fix up some CSS and will check the other 2 comments then.

**Pull request merged**